.github/workflows/build.yml

```diff
@@ -24,6 +24,8 @@ jobs:
       run: chmod +x gradlew
     - name: Build with Gradle
       run: ./gradlew build --stacktrace
+    - name: Check code style
+      run: ./gradlew detekt
```

Comment:
Having `./gradlew check` would be better, since, it would run all checks like tests(when they are written) and detekt and other checks as well.